binder/postBuild

```diff
@@ -37,11 +37,18 @@ jupyter labextension link --no-build packages/application/
 jupyter labextension link --no-build packages/ui-components/
 jupyter labextension link --no-build packages/metadata-common/
 jupyter lab build --dev-build=False --minimize=False --debug
+
+pwd
```

Comment:
Is this for troubleshooting?  A comment to that effect might be helpful.

Reply:
Yes it was. I removed it because it can be easily re-added.